config/webhook/webhook_patch.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: validating.odh-model-controller.opendatahub.io
+  annotations:
+    service.beta.openshift.io/inject-cabundle: true
+webhooks:
+  - name: validating.ksvc.odh-model-controller.opendatahub.io
+    clientConfig:
+      service:
+        name: odh-model-controller-webhook-service
```

Comment:
May the patch include the namespace?

Running `kustomize build config/base` on the `main` branch, then on this PR, and doing a diff of both results shows that the `clientConfig` keeps `system` namespace, which would be a bug:

![image](https://github.com/opendatahub-io/odh-model-controller/assets/23639005/1172cfcc-00ce-4d15-9e6d-cea7c13c25be)

Reply:
Hmm, yesterday, we talked about it and you suggested not to patch the namespace as it would be replaced anyway.
I can revert back to the patch I have added.

Reply:
From the cluster:
```yaml
    service:
      name: odh-model-controller-webhook-service
      namespace: redhat-ods-applications
      path: /validate-serving-knative-dev-v1-service
      port: 443
  failurePolicy: Fail
  matchPolicy: Equivalent
  name: validating.ksvc.odh-model-controller.opendatahub.io
  namespaceSelector: {}
  objectSelector:
    matchExpressions:
    - key: serving.kserve.io/inferenceservice
      operator: Exists

```

Reply:
OK, I just tested with odh-operator and the result is the expected.
It looks like the odh-operator is doing some additional processing than kustomize.